code/modules/crafting/recipes.dm

```diff
@@ -9,6 +9,13 @@
 	var/chem_catalysts[] = list() //like tools but for reagents
 	var/category = CAT_NONE //where it shows up in the crafting UI
 
+/datum/crafting_recipe/spear
+	name = "Spear"
+	result = /obj/item/weapon/twohanded/spear
+	reqs = list(/obj/item/weapon/restraints/handcuffs/cable = 1,
+				/obj/item/stack/rods = 1)
```

Comment:
While I dislike this PR....you kinda made spears slightly easier to craft. You forgot the glass shard in the recipe. Honk.